asgi.py

```diff
@@ -0,0 +1,19 @@
+import uvicorn
+
+from src.app.environment import get_settings
+from src.app.log import CONFIG as LOGGING_CONFIG
+
+settings = get_settings()
+
+
+if __name__ == "__main__":
+    server = uvicorn.Server(
+        uvicorn.Config(
```

Comment:
I used this `Server(Config())` approach because I found that when using just `uvicorn.run()`, some exception was raised when trying to set `reload` and `log_config` simultaneously.